src/ApiService/ApiService/onefuzzlib/notifications/Ado.cs

```diff
@@ -24,7 +24,10 @@ public class Ado : NotificationsBase, IAdo {
         Report? report;
         if (reportable is RegressionReport regressionReport) {
             if (regressionReport.CrashTestResult.CrashReport is not null) {
-                report = regressionReport.CrashTestResult.CrashReport;
+                report = regressionReport.CrashTestResult.CrashReport with {
+                    InputBlob = regressionReport?.OriginalCrashTestResult?.CrashReport?.InputBlob ??
```

Comment:
the first `?` is not needed in this expression. `regressionReport` is already not null